README.md

```diff
@@ -87,15 +162,27 @@ Endpoints made available by server are specified [here](./web/swagger/api.yaml).
      # Optionally, one can specify collateral in the configuration to avoid sending it's information in the endpoints which require it.
     collateral: 15522d2518b36bdeb8e2e4829aff7ae9e5afbf74387d756543c5e955e83a9434#2
     ```
-3. Run the server with command `cabal run geniusyield-server -- serve -c my-config.yaml`. Run `cabal run geniusyield-server -- -h` for help 😉.
-4. Test if server is running successfully by calling, say, `/settings` endpoint. Example `curl` request: `curl -H 'api-key: YOUR_SECRET_KEY' -X GET http://localhost:8082/v0/settings | jq`, assuming port was specified as `8082`. On success, it should return something akin to:
-    ```json
-    {
-      "network": "mainnet",
-      "version": "0.1.0",
-      "revision": "c2f8db2bc82a13c850c3b0088a1ce089bb1065b7",
-      "backend": "mmb"
-    }
+3. Run the server with command `cabal run geniusyield-server -- serve -c my-config.yaml`
+4. Run `cabal run geniusyield-server -- -h` for help 😉
```

Comment:
This is not really the fourth step and was given as a remark for third step.

Reply:
I copied over this from your original docs, but I can change it if you think that it is better to have it differently.